### PR TITLE
Feat/add db module

### PR DESCRIPTION
**Pull request by eldimious**

### CODE REVIEW

File: backend/apis/services/usersService/src/index.js [hunk 1]
```diff
@@ -3,19 +3,36 @@ const cors = require('cors');
...
   makeGetRequest
 } = require('./common/utils');
 const users = require('./common/users');
 
+const PRODUCTION_ENV = 'production';
+const defaultOptions = {
+  dialect: 'postgres',
+  logging: true,
+  timezone: '+00:00',
+  dialectOptions: process.env.NODE_ENV === PRODUCTION_ENV
```
Comment (akoufa) [inline backend/apis/services/usersService/src/index.js:21]: This can be hidden in a method `isProduction`

File: devops/aws/ecs_ec2/main.tf [hunk 2]
```diff
@@ -316,6 +411,26 @@ module "ecs_users_api_ec2" {
...
       "name" : "RECOMMENDATIONS_SERVICE_URL",
       "value" : "http://${module.ecs_recommendations_api_ec2.aws_service_discovery_service_name}.${aws_service_discovery_private_dns_namespace.segment.name}:${var.recommendations_api_port}"
+    },
+    {
+      "name" : "POSTGRES_USER",
+      "value" : "${tostring(module.users_database.db_instance_username)}",
+    },
+    {
+      "name" : "POSTGRES_PASSWORD",
+      "value" : "${tostring(module.users_database.db_master_password)}",
```
Comment (akoufa) [inline devops/aws/ecs_ec2/main.tf:421]: Is this security wise ok to define the password in terraform ?

Comment (eldimious) [reply devops/aws/ecs_ec2/main.tf]: I used `secrets` on task definition according to https://www.chakray.com/creating-fargate-ecs-task-aws-using-terraform/ and https://blog.gruntwork.io/a-comprehensive-guide-to-managing-secrets-in-your-terraform-code-1d586955ace1#bebe using aws_secretsmanager_secret resource.
We can create an issue to investigate it more if u want

Comment (eldimious) [reply devops/aws/ecs_ec2/main.tf]: https://github.com/arconsis/aws-network-microservices-warmup/issues/85

**Review by akoufa — APPROVED** [review]

**Pull request merged**